### PR TITLE
Updating 'entity' doc with TTL & general improvements - DOC-7191

### PR DESCRIPTION
**Pull request by zuluecho9**
See DOC-7191 for details. Synopsis of changes: 

* Added links to the github repo where our entity definitions live and explained what info is there.
* Made some general improvements, to help with flow of doc (links to different section and such), and to improve description of what an entity is to reflect current functionality.

### CODE REVIEW

File: src/content/docs/new-relic-one/use-new-relic-one/core-concepts/what-entity-new-relic.mdx [hunk 1]
```diff
@@ -13,50 +13,57 @@ redirects:
   - /docs/what-entity-new-relic
 ---
 
-New Relic monitoring is built around the concept of the **entity**. An entity is anything that reports data to New Relic. This document explains:
+New Relic monitoring is built around the concept of **entities**, which are basically the hosts and services and other systems that New Relic monitors. In this doc, you'll learn how we define entities, what you can do with them, and how you can create your own entities or groupings of entities.
```
Comment (urbiz-nr) [inline src/content/docs/new-relic-one/use-new-relic-one/core-concepts/what-entity-new-relic.mdx:16]: I prefer the original phrasing. We like to emphasize the fact that it's literally *anything* that reports data, we don't want to start the doc by narrowing it down to existing components. Just my opinion, obviously.

File: src/content/docs/new-relic-one/use-new-relic-one/core-concepts/what-entity-new-relic.mdx [hunk 2]
```diff
@@ -13,50 +13,57 @@ redirects:
...
 
-"Entity" can refer to fundamental data-reporting components like applications, hosts, and database services, but it can also refer to larger groupings of those components. For example, to monitor a data center, you could aggregate those hosts in New Relic to be a [workload](/docs/new-relic-one/use-new-relic-one/get-started/introduction-new-relic-one) (a custom grouping of entities). That workload is, itself, also an entity.
+Also very important is the [relationships between entities](/docs/apis/nerdgraph/examples/nerdgraph-relationships-api-tutorial). Our behind-the-scenes relationship-mapping helps us understand how entities are connected, how they affect each other. And this allows us to give you the power to configure how any data you're bringing in is related to existing entities, or how it's related to other entities.
```
Comment (urbiz-nr) [inline src/content/docs/new-relic-one/use-new-relic-one/core-concepts/what-entity-new-relic.mdx:24]: Nice!

File: src/content/docs/new-relic-one/use-new-relic-one/core-concepts/what-entity-new-relic.mdx [hunk 3]
```diff
@@ -13,50 +13,57 @@ redirects:
...
-<Callout variant="tip">
-  For more information on how to modify existing entity types or create new ones please refer to our [Entity Synthesis documentation](https://github.com/newrelic/entity-definitions#entity-definitions).
-</Callout>
+## Configure your own entity data with entity synthesis [#entity-synthesis]
+
+If you have telemetry from any source that's not supported by New Relic out of the box, you can propose a mapping for it. Once approved, any telemetry received by New Relic that matches your definition file will be synthesized into an entity. 
 
+To learn more: 
+* To learn about reserved attributes and how entity relationships are defined, keep reading this doc. 
+* To learn how to do the work of modifying existing entity types or creating new ones, see [our GitHub repo on entity synthesis](https://github.com/newrelic/entity-definitions#entity-definitions).
```
Comment (urbiz-nr) [inline src/content/docs/new-relic-one/use-new-relic-one/core-concepts/what-entity-new-relic.mdx:62]: The possibility of creating entities is huge, I think it needs to stand out.

File: src/content/docs/new-relic-one/use-new-relic-one/core-concepts/what-entity-new-relic.mdx [hunk 4]
```diff
@@ -13,50 +13,57 @@ redirects:
...
+* [Create workloads](/docs/new-relic-one/use-new-relic-one/workloads/workloads-new-relic-one-isolate-resolve-incidents-faster), which allow you to group business-important sets of entities. 
 
-<Callout variant="tip">
-  For more information on how to modify existing entity types or create new ones please refer to our [Entity Synthesis documentation](https://github.com/newrelic/entity-definitions#entity-definitions).
-</Callout>
+## Configure your own entity data with entity synthesis [#entity-synthesis]
+
+If you have telemetry from any source that's not supported by New Relic out of the box, you can propose a mapping for it. Once approved, any telemetry received by New Relic that matches your definition file will be synthesized into an entity. 
 
+To learn more:
```
Comment (urbiz-nr) [inline src/content/docs/new-relic-one/use-new-relic-one/core-concepts/what-entity-new-relic.mdx:60]: Too many "learn" here.

File: src/content/docs/new-relic-one/use-new-relic-one/core-concepts/what-entity-new-relic.mdx [hunk 5]
```diff
@@ -13,50 +13,57 @@ redirects:
...
+* For most entities, its GUID is reported as the attribute [`entityGuid`](/?event=SyntheticCheck&attribute=entityGuid). For workloads, it's `workloadGuid`. You can run [NRQL queries](/docs/query-your-data/nrql-new-relic-query-language/get-started/introduction-nrql-new-relics-query-language) to find entities by their GUID.
+* To see connections between entities, you have several options: 
+  * When viewing a specific entity in the UI, use the [**Related entities** UI](#related-entities). 
+  * Use [service maps](/docs/service-maps-dependencies-new-relic-one). 
+  * Use [distributed tracing](/docs/distributed-tracing-new-relic-one). 
+  * Use [our NerdGraph API to view entity relationships](/docs/apis/nerdgraph/examples/nerdgraph-relationships-api-tutorial).
+* To group entities together, see [Group entities](#group). 
+* [Configure entity definitions and relationships.](#entity-synthesis)
+* To learn technical details about entity types, see our [GitHub repo](https://github.com/newrelic/entity-definitions/tree/main/definitions). In an entity type's `definition` file, you'll see information like: 
+  * The `domain`: for example, `APM`, or `Infra`.
```
Comment (urbiz-nr) [inline src/content/docs/new-relic-one/use-new-relic-one/core-concepts/what-entity-new-relic.mdx:42]: Block letter after ":" .

File: src/content/docs/new-relic-one/use-new-relic-one/core-concepts/what-entity-new-relic.mdx [hunk 6]
```diff
@@ -13,50 +13,57 @@ redirects:
...
+Some tips for finding and understanding entity data:
 
-<Callout variant="tip">
-  You can create new entity types to monitor any data source. Learn more about [entity synthesis](https://github.com/newrelic/entity-definitions#entity-definitions).
-</Callout>
+* To find an entity's `entityGuid` and `entityName` and other metadata: from any list of monitored entities in the [New Relic Explorer](/docs/new-relic-one/use-new-relic-one/ui-data/new-relic-one-entity-explorer-view-performance-across-apps-services-hosts), click an entity's <Icon name="fe-more-horizontal"/> icon, and click **See metadata and tags**.
+* For most entities, its GUID is reported as the attribute [`entityGuid`](/?event=SyntheticCheck&attribute=entityGuid). For workloads, it's `workloadGuid`. You can run [NRQL queries](/docs/query-your-data/nrql-new-relic-query-language/get-started/introduction-nrql-new-relics-query-language) to find entities by their GUID.
+* To see connections between entities, you have several options: 
+  * When viewing a specific entity in the UI, use the [**Related entities** UI](#related-entities). 
+  * Use [service maps](/docs/service-maps-dependencies-new-relic-one).
```
Comment (urbiz-nr) [inline src/content/docs/new-relic-one/use-new-relic-one/core-concepts/what-entity-new-relic.mdx:36]: Too many "use". I'd remove them and leave only the feature.

File: src/content/docs/new-relic-one/use-new-relic-one/core-concepts/what-entity-new-relic.mdx (hunk 1 above)
Comment (zuluecho9) [reply src/content/docs/new-relic-one/use-new-relic-one/core-concepts/what-entity-new-relic.mdx:16]: it's a little more vague than even that, though, which is part of the problem I was having. For example: an entity can be a dashboard or workload (and soon some other things, like a github repo), which aren't technically things that report data to New Relic. So I was trying to find a way to say 'it's basically that' even though it's not. But I"ll work on it some more.

Comment (urbiz-nr) [reply src/content/docs/new-relic-one/use-new-relic-one/core-concepts/what-entity-new-relic.mdx:16]: You're right. How about "An entity is anything that generates data and New Relic can monitor".

File: src/content/docs/new-relic-one/use-new-relic-one/core-concepts/what-entity-new-relic.mdx (hunk 5 above)
Comment (zuluecho9) [reply src/content/docs/new-relic-one/use-new-relic-one/core-concepts/what-entity-new-relic.mdx:42]: Sorry, I'm not sure what that one means

Comment (urbiz-nr) [reply src/content/docs/new-relic-one/use-new-relic-one/core-concepts/what-entity-new-relic.mdx:42]: We write the word in capitals after the :

Comment (urbiz-nr) [reply src/content/docs/new-relic-one/use-new-relic-one/core-concepts/what-entity-new-relic.mdx:42]: The `domain`: For example...

Comment (zuluecho9) [reply src/content/docs/new-relic-one/use-new-relic-one/core-concepts/what-entity-new-relic.mdx:42]: oh i see. i was looking at wrong thing

**Pull request merged**